galley/pkg/config/analysis/analyzers/virtualservice/destinationhosts.go

```diff
@@ -157,7 +156,8 @@ func initServiceEntryHostMap(ctx analysis.Context) map[util.ScopedFqdn]*v1alpha3
 			Hosts: []string{host},
 			Ports: ports,
 		}
-		result[util.NewScopedFqdn(hostsNamespaceScope, r.Metadata.FullName.Namespace, r.Metadata.FullName.Name.String())] = se
+		// k8s Services export to all namespaces, they have no "exportTo" concept
```

Comment:
Actually it does! based on annotation: https://istio.io/docs/reference/config/annotations/ `networking.istio.io/exportTo`